### ISSUE TITLE
Move node bindings and node loader out of webidl folder

### ISSUE THREAD

**Issue by brendandahl** [mozilla/positron#140]
They don't use webidl.

**Comment by Ryuno-Ki** [comment]
Where does WebIDL stands for? [Web Interface Description Language](https://developer.mozilla.org/en-US/docs/Glossary/WebIDL)?

**Comment by mykmelez** [comment]
> Where does WebIDL stands for? Web Interface Description Language?

Yes, that's right. We landed the Node integration bits into the positron/webidl/ subdirectory, but they don't actually use WebIDL, so we should move them to a directory with a different name.

**Comment by Ryuno-Ki** [comment]
Ah, the wiki permission were granted over in SpiderNode >_<

Cannot edit/create a glossary here. No permissions.

**Comment by mykmelez** [comment]
> Ah, the wiki permission were granted over in SpiderNode >_<
> 
> Cannot edit/create a glossary here. No permissions.

Fixed! You should have the ability to do so now.
